tir/technologies/webapp_internal.py

```diff
@@ -6610,7 +6610,7 @@ def fill_grid(self, field, x3_dictionaries, initial_layer, duplicate_fields=[]):
                                                     self.send_keys(selenium_input(), Keys.ENTER)
 
                                     elif lenfield == len(field[1]) and self.get_current_container().attrs[
-                                        'id'] != container_id:
+                                        'id'] != container_id and not modal_open:
```

Comment:
```suggestion
                                        'id'] != container_id:
```

remover a condição pois impacta nos campos que preenchem e nao fecham sozinho